src/ngAnimate/animateCss.js

```diff
@@ -324,6 +325,23 @@ function createLocalCacheLookup() {
   };
 }
 
+function registerRestorableStyles(backup, node, properties) {
+  forEach(properties, function(prop) {
+    // we do not reassign an already present style value since
+    // if we detect the style property value again we may be
+    // detecting styles that were added via the `from` styles.
```

Comment:
I would move this first sentence into a comment above the function declaration since it is about what this function is doing in general, whereas the following sentences are about the implementation issue within the function.